--- newlib-1.18.0/newlib/libc/include/stdlib.h	2009-12-17 14:43:43.000000000 -0500
***************
*** 101,112 ****
  char *	_EXFUN(mkdtemp,(char *));
  int	_EXFUN(mkstemp,(char *));
  int	_EXFUN(mkstemps,(char *, int));
! char *	_EXFUN(mktemp,(char *) _ATTRIBUTE ((warning ("the use of `mktemp' is dangerous; use `mkstemp' instead"))));
  #endif
  char *	_EXFUN(_mkdtemp_r, (struct _reent *, char *));
  int	_EXFUN(_mkstemp_r, (struct _reent *, char *));
  int	_EXFUN(_mkstemps_r, (struct _reent *, char *, int));
! char *	_EXFUN(_mktemp_r, (struct _reent *, char *) _ATTRIBUTE ((warning ("the use of `mktemp' is dangerous; use `mkstemp' instead"))));
  #endif
  _VOID	_EXFUN(qsort,(_PTR __base, size_t __nmemb, size_t __size, int(*_compar)(const _PTR, const _PTR)));
  int	_EXFUN(rand,(_VOID));
--- 101,112 ----
  char *	_EXFUN(mkdtemp,(char *));
  int	_EXFUN(mkstemp,(char *));
  int	_EXFUN(mkstemps,(char *, int));
! char *	_EXFUN(mktemp,(char *));
  #endif
  char *	_EXFUN(_mkdtemp_r, (struct _reent *, char *));
  int	_EXFUN(_mkstemp_r, (struct _reent *, char *));
  int	_EXFUN(_mkstemps_r, (struct _reent *, char *, int));
! char *	_EXFUN(_mktemp_r, (struct _reent *, char *));
  #endif
  _VOID	_EXFUN(qsort,(_PTR __base, size_t __nmemb, size_t __size, int(*_compar)(const _PTR, const _PTR)));
  int	_EXFUN(rand,(_VOID));
--- newlib-1.18.0/newlib/libm/common/s_infconst.c	2009-05-15 12:15:57.000000000 -0400
***************
*** 14,21 ****
  
  /* Double version of infinity.  */
  const union __dmath __infinity[1] = { { DBL_MAX+DBL_MAX } };
- 
- /* Long double version of infinity.  */
- #if defined(_HAVE_LONG_DOUBLE)
- const union __ldmath __infinityld[1] = { { LDBL_MAX+LDBL_MAX } };
- #endif
--- 14,16 ----
